lib/openstack_api/openstack_floating_ip.py

```diff
@@ -1,21 +1,18 @@
-from typing import List, Dict
+from typing import List, Dict, Callable, Any
 
 from openstack.network.v2.floating_ip import FloatingIP
 from openstack_api.dataclasses import (
     NonExistentCheckParams,
     NonExistentProjectCheckParams,
+    EmailQueryParams,
 )
 
 from openstack_api.openstack_connection import OpenstackConnection
+from openstack_api.openstack_query_email_base import OpenstackQueryEmailBase
 from openstack_api.openstack_identity import OpenstackIdentity
-from openstack_api.openstack_query import OpenstackQuery
-from openstack_api.openstack_wrapper_base import OpenstackWrapperBase
-from openstack_api.dataclasses import EmailQueryParams
-from structs.email_params import EmailParams
-from structs.smtp_account import SMTPAccount
 
 
-class OpenstackFloatingIP(OpenstackWrapperBase):
+class OpenstackFloatingIP(OpenstackQueryEmailBase):
```

Comment:
Inherit from both classes rather instead of a single one

That should it's an action and it handles emails too